db/structure.sql

```diff
@@ -2,12 +2,16 @@
 -- PostgreSQL database dump
```

Comment:
Yea, Orientation uses some special database foo that requires you to dump your schema like this,